src/agent/adu_core_interface/tests/main.cpp

```diff
@@ -13,12 +13,6 @@
 
 #include <ostream>
 
-#ifndef ENABLE_MOCKS
-
-#    error "ENABLE_MOCKS must be defined to build UTs."
-
-#endif
```

Comment:
not needed anymore?

Reply:
Does this still exist in this iter? It doesn't look like it does for me.

---
In reply to: [591975381](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591975381) [](ancestors = 591975381)